### PR TITLE
Node electron

### PR DESCRIPTION
**Pull request by BenTheKlein**

### CODE REVIEW

File: node-electron/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+2. Adding a require statement to the project entry file:
+    ```js
+    const rook = require("rookout/auto_start");
+    ```
+
+3. Update the command line for electron to contain --inspect, In the `package.json`:
+	```
+	"scripts": {
+    "start": "electron . --inspect" },
```
Comment (MickaelAlliel) [inline node-electron/README.md]: Add new line before closing bracket }

File: node-electron/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+For more information about Docker go [here](https://www.docker.com/).
+
+
+
+## Rookout Integration Explained
+
+There are 4 simple steps to integrate Rookout into your existing Node-Electron application for an [agentless] setup:
+
+1. Adding rookout to `package.json` available on [npm]:
```
Comment (MickaelAlliel) [inline node-electron/README.md]: You added [npm] link but no reference to the link is in that README. Please add to the bottom the link like the other one you'll find at the bottom

File: node-electron/README.md (hunk 1 above)
Comment (BenTheKlein) [reply node-electron/README.md]: fixed it

**Comment by MickaelAlliel** [comment]
Please see other comments

**Comment by BenTheKlein** [comment]
@MickaelAlliel  Done!

**Comment by BenTheKlein** [comment]
@MickaelAlliel - can you approve this?

**Comment by BenTheKlein** [comment]
Fixed Rookout/Rookout#1302

**Pull request merged**